### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stong**

### CODE REVIEW

**Review by alystair — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The directories also have no explicit field for parent directories, so one can l
 This allows for graph structures inside a file system, which a DFA basically is.
 
 ### Q: Should I use this <del>in production</del> anywhere?
-A: No, but I can't stop you.
+A: Yes
```
Comment (8051Enthusiast) [inline README.md:29]: Pull request looks mostly fine, but I'm not quite happy with this line change.
The version of this program is only 0.1.0, which by SemVer can not be considered stable.
It's also quite a big change to the documentation.

**Review by 8051Enthusiast — CHANGES_REQUESTED** [review]
I think this change requires an RFC and careful consideration before it could be merged.

File: README.md (hunk 1 above)
Comment (stong) [reply README.md:29]: Fixed in new commit

**Comment by stong** [comment]
Hi, I have addressed the concerns in the new commit. Thanks

**Pull request closed without merging**